test/integration/model/update.test.js

```diff
@@ -118,7 +118,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       expect(account.ownerId).to.be.equal(ownerId);
     });
 
-    if (_.get(current.dialect.supports, 'returnValues.returning')) {
+    if (current.dialect.supports.returnValues === 'returning') {
```

Comment:
These were the only usages of lodash so that import can be removed

Reply:
Done